mayhem/Mayhemfile

```diff
@@ -0,0 +1,9 @@
+project: pdfplumber
```

Comment:
Is the timeout required to be 8?

Reply:
The defect it found took ~4.5 seconds, so I'll drop it to 6 to be safe. The PDF parsing is fairly expensive at its worst case